docker-compose.yml

```diff
@@ -58,6 +64,29 @@ services:
       - redisinsight:/db
     ports:
       - 8001:8001
+  localstack:
+    image: localstack/localstack
+    ports:
```

Comment:
I think the only port we need to expose is 4566, and a comment saying we expose that port so that you can use the s3 client to talk to the localstack instance